helm/dashboards/charts/private_dashboards_mz/dashboards/shared/private/team-firecracker.json

```diff
@@ -23,7 +23,7 @@
       "bars": false,
       "dashLength": 10,
       "dashes": false,
```

Comment:
@weseven I think you probably have a better one with the blackbox exporter now right?